Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -4136,6 +4136,9 @@ bool CStaticFunctionDefinitions::WarpPedIntoVehicle(CPed* pPed, CVehicle* pVehic
     assert(pPed);
     assert(pVehicle);
 
+    if(pVehicle->GetMaxPassengers() == VEHICLE_PASSENGERS_UNDEFINED)
```

Comment:
`VEHICLE_PASSENGERS_UNDEFINED` = 255;
`GetMaxPassengers()` returns 255 if the seat amount is not defined **or** if the vehicle is not drive-able (like trailers)